doc/source/rllib/core-concepts.rst

```diff
@@ -1,13 +1,67 @@
+.. include:: /_includes/rllib_announcement.rst
 
-.. TODO: Deprecate this page (moved here from the old index (rllib in 60s) page).
+.. include:: /_includes/rllib_we_are_hiring.rst
+
+.. TODO: We need trainers, environments, algorithms, policies, models here. Likely in that order.
```

Comment:
Good point. Envs should probably be explained before Trainers, even.

Reply:
It's very difficult. Conceptually, envs and algos come before / feed into trainer. From a "getting started" perspective, trainers should go first.